Formula/molten-vk.rb

```diff
@@ -143,6 +144,9 @@ def install
     include.install Dir["Package/Release/MoltenVKShaderConverter/include/" \
                         "MoltenVKShaderConverter"]
 
+    inreplace "MoltenVK/icd/MoltenVK_icd.json",
+              "./libMoltenVK.dylib",
+              (lib/"libMoltenVK.dylib").relative_path_from(share/"vulkan/icd.d")
```

Comment:
I just noticed that this change seems to be completely independent of `vulkan-loader`. Is that correct? If so, please move this change into a separate PR.

Reply:
Neither _technically_ depends on the other (e.g. on linux, you'd want `vulkan-loader` but can't have `molten-vk`, and on macOS, it's entirely legal to just link directly against `libMoltenVK.dylib` and skip `vulkan-loader` altogether). But `vulkan-loader`'s purpose is to dynamically load and dispatch to Vulkan ICDs at runtime, and `molten-vk` is the Vulkan ICD for macOS, so the `vulkan-loader` package on macOS is useless (and can't have its functionality properly tested) without a working `molten-vk` ICD registry JSON file. Meanwhile, on Linux, you'd use it to load a different driver (e.g. mesa). On the other hand, the ICD registry JSON file is only useful when using `vulkan-loader`.

I can move this to a separate PR if you'd prefer, but they're in a sort of "neither can be tested without the other" situation, so I figured it made sense to bundle them.

Reply:
Do you need `molten-vk` to be installed when testing `vulkan-loader, at least on macOS? Because it won't be installed unless you declare a dependency.

Reply:
`brew test vulkan-loader` will work fine without it, but more complex functional testing (i.e. something that uses Vulkan to draw to a window) wouldn't. So you can test that the package is linked properly, but not actually _use_ it for anything. (And I don't think it'd be appropriate to have the test program try to draw something, since (b) it'd be absurdly complex, and (b) it wouldn't work anywhere where a suitable driver isn't necessarily available, e.g. potentially linux).